### ISSUE TITLE
Hard dependency on librosa because of package level import

### ISSUE THREAD

**Issue by chiragjn** [makcedward/nlpaug#62]
Minor issue, but a hard dependency on librosa still exists but is not mentioned in dependencies
```
|     import nlpaug.augmenter.char as nac
|   File "/usr/local/lib/python3.6/site-packages/nlpaug/__init__.py", line 2, in <module>
|     from nlpaug.base_augmenter import *
|   File "/usr/local/lib/python3.6/site-packages/nlpaug/base_augmenter.py", line 5, in <module>
|     from nlpaug.util import Action, Method, WarningException, WarningName, WarningCode, WarningMessage
|   File "/usr/local/lib/python3.6/site-packages/nlpaug/util/__init__.py", line 7, in <module>
|     from nlpaug.util.visual import *
|   File "/usr/local/lib/python3.6/site-packages/nlpaug/util/visual/__init__.py", line 1, in <module>
|     from nlpaug.util.visual.spectrogram import *
|   File "/usr/local/lib/python3.6/site-packages/nlpaug/util/visual/spectrogram.py", line 2, in <module>
|     import librosa.display
| ModuleNotFoundError: No module named 'librosa'
```

**Comment by makcedward** [comment]
Will remove hard dependency in next release.

**Comment by makcedward** [comment]
Included this fix in 0.0.11 release.
